ts/parsers/shapeparser.ts

```diff
@@ -1,4 +1,4 @@
-import { CheckValidObject as checkPath } from "../helpers/checkobj";
```

Comment:
any particular reason why this doesn't work for you? Seem to me a like a better and descriptive alias

Reply:
![image](https://user-images.githubusercontent.com/16796013/123274109-46463300-d514-11eb-9ad2-5f344f1b9847.png)


Well it automatically got imported like that when used the function. 
@kryasaleg noticed duplicate import later on so i removed the one.

Anyway i added the alias back.